packages/core/src/2d/text/Font.ts

```diff
@@ -0,0 +1,50 @@
+import { RefObject } from "../../asset/RefObject"
+import { assignmentClone } from "../../clone/CloneManager";
+import { Engine } from "../../Engine";
+import { UpdateFlag } from "../../UpdateFlag";
+import { UpdateFlagManager } from "../../UpdateFlagManager";
+
+export class Font extends RefObject {
+  @assignmentClone
+  private _fontName: string = "Arial";
+
+  private _updateFlagManager: UpdateFlagManager = new UpdateFlagManager();
+
+  /**
+   * The font name of the TextRenderer.
+   */
+  get fontName(): string {
+    return this._fontName;
+  }
+
+  set fontName(value: string) {
+    value = value || "Arial";
+    if (this._fontName !== value) {
+      this._fontName = value;
+      this._updateFlagManager.distribute();
+    }
+  }
```

Comment:
The class already named `Font`, this property use name is better.

Reply:
done